### PR TITLE
Adding radar

### PR DESCRIPTION
**Pull request by glennwester**
This is something that was requested

### CODE REVIEW

**Comment by glennwester** [comment]
Cool!

File: XWeather/UITests/AppInitializer.cs [hunk 1]
```diff
@@ -16,6 +16,8 @@ public static IApp StartApp (Platform platform)
 
 			//this is an important change! 
 
+			//adding radar functionality
```
Comment (glennwester) [inline XWeather/UITests/AppInitializer.cs:19]: hmm this looks interesting

Comment (glennwester) [reply XWeather/UITests/AppInitializer.cs:19]: agreed

Comment (glennwester) [reply XWeather/UITests/AppInitializer.cs:19]: Perhaps fix this up

Comment (glennwester) [reply XWeather/UITests/AppInitializer.cs:19]: Looks good

**Review by glennwester — COMMENTED** [review]
Great

File: XWeather/UITests/AppInitializer.cs (hunk 1 above)
Comment (OpticksVision) [inline XWeather/UITests/AppInitializer.cs:19]: dsvs

Comment (OpticksVision) [reply XWeather/UITests/AppInitializer.cs:19]: dsfsdfa

**Review by OpticksVision — CHANGES_REQUESTED** [review]
Go back

**Pull request merged**